packages/jest-preset-mc-app/setup-test-framework.js

```diff
@@ -1,13 +1,13 @@
-require('unfetch/polyfill');
-require('jest-enzyme');
-require('@testing-library/jest-dom/extend-expect');
-require('@testing-library/react/cleanup-after-each');
-const Enzyme = require('enzyme');
-const Adapter = require('enzyme-adapter-react-16');
-const ShallowWrapper = require('enzyme/ShallowWrapper');
-const configureEnzymeExtensions = require('@commercetools/enzyme-extensions');
-const commerceToolsEnzymeMatchers = require('@commercetools/jest-enzyme-matchers');
-require('./polyfills/intl');
+import 'unfetch/polyfill';
+import 'jest-enzyme';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Using `import` is actually an attempt to instruct vscode to look up the extended types of the custom matchers.

We'll have to look into this more as a follow up